src/constants/selectors.js

```diff
@@ -0,0 +1 @@
+export const userInfoSelector = (state) => state.userInfo;
```

Comment:
I cannot see the usage of the selector :(
And btw, selectors are not constants, move it on the same level with constants